sdk/containerregistry/azure-containerregistry/azure/containerregistry/_generated/models/_models.py

```diff
@@ -1127,6 +1095,44 @@ def __init__(
         self.schema_version = schema_version
 
 
+class OCIIndex(Manifest):
```

Comment:
Is this part of the public API or is it hidden?  We've chosen not to expose this publicly for the time being, but may add it in future releases.  We should have issues tracking it in our backlogs like this one: https://github.com/Azure/azure-sdk-for-net/issues/33647

Reply:
- This model is hidden. Btw, our published model is listing here: https://github.com/Azure/azure-sdk-for-python/blob/3c878d86505652b103be57dfee7160d9c0887076/sdk/containerregistry/azure-containerregistry/azure/containerregistry/__init__.py
- Thanks for letting me know, created one for Python: 
 https://github.com/Azure/azure-sdk-for-python/issues/29576

Reply:
Thank you!